crates/bevy_pbr/src/render/rgb9e5.wgsl

```diff
@@ -44,20 +44,22 @@ fn vec3_to_rgb9e5_(rgb_in: vec3<f32>) -> u32 {
     return (u32(exp_shared) << 27u) | (n.b << 18u) | (n.g << 9u) | (n.r << 0u);
 }
 
-#ifdef WEBGL
-fn extractBits(value: u32, offset: u32, bits: u32) -> u32 {
+// #ifdef WEBGL
+// Builtin extractBits() is not working on WEBGL or DX12
```

Comment:
Atl for DX12, depends on wgpu 0.17. Idk about WebGL...

Reply:
https://registry.khronos.org/OpenGL-Refpages/es3/html/bitfieldExtract.xhtml

The keyword is supported in gles 3.1, but not gles 3.0, and afaik webgl2 is based on gles 3.0.